gui/src/widgets/pagenavigationwidget.cpp

```diff
@@ -0,0 +1,82 @@
+#include "widgets/pagenavigationwidget.h"
+#include <QBoxLayout>
+#include <QDebug>
+#include <QLoggingCategory>
+#include <stylehelper.h>
+
+using namespace scopy;
+Q_LOGGING_CATEGORY(CAT_NAVIGATIONWIDGET,"PageNavigationWidget")
+
+PageNavigationWidget::PageNavigationWidget(bool hasHome, bool hasOpen, QWidget *parent)
+	: QWidget(parent)
+{
+	initUI(parent);
+
+	hasHomeBtn(hasHome);
```

Comment:
hasHomeBtn is a confusing name for a setter. Consider renaming this to "setHomeBtnVisible"